nix/machines/monitor.nix

```diff
@@ -65,11 +66,18 @@ in
         datasources.settings.datasources = [
           {
             name = "prometheus";
+            uid  = "P1809F7CD0C75ACF3";
```

Comment:
just to clarify, does this value have any specific meaning or does it just need to be $SOME_VALUE that matches all the datasource references form the dashboard json?

Reply:
Im unclear on this myself

@jshcmpbll might be able to add some insight

Reply:
@kylerisse Yup! The value itself has no real meaning but would need to match the data source values found in the dashboard like so:

https://github.com/socallinuxexpo/scale-network/blob/9258aee2c9590d4bfe924cad478c6c2b6debbe05/monitoring/openwrt_dashboard.json#L72

Each time I started the VM it seemed to use that UID for the datasource by default but I figured we might as well set it so there isn't a possibility of some change in the future resulting in the UID changing.

Reply:
sweet, thanks @jshcmpbll